doc/dev/data-structures.md

```diff
@@ -369,34 +369,29 @@ All structs are defined in [opmphm.h](/src/include/kdbopmphm.h).
 The OPMPHM is not an ordinary hash map. The OPMPHM inserts all elements at once,
 this process is known as build. The OPMPHM is designed to return an index when looking up
 an element with `opmphmLookup ()`. Those indices must be known when building the OPMPHM.
-Due to the compression of the OPMPHM data structure, adding an element will most likely
-lead to a complete rebuild.
 
-The index, also known as the order is set in `OpmphmOrder->index.p`,
-additionally a minimum and maximum order is needed in `OpmphmInit->minOrder`
-and `OpmphmInit->maxOrder`.
+The desired return index, also known as the order is set in `OpmphmGraph->edges[i].order`.
+Where `i` is the i-th element.
 
 The build consists of three steps.
 
-### The Initialization
+### Initialization
 
+Use `opmphmNew ()` and `opmphmGraphNew ()` to instantiate the needed structures.
+To initialize the OPMPHM build the `OpmphmInit` must be set with information about your data.
 Set your data array `OpmphmInit->data` and the string extraction function `OpmphmInit->getString`,
 which should extract the string from a single data array entry.
 Provide a good seed in `OpmphmInit->initSeed`, needed in the next step.
-Call the `opmphmNewOrder ()` and use the default order or your own order.
-Min and Max order also need to be set respectively.
 
-The `opmphmInit ()`, transforms the orders in a internal representation, this step
-is irreversible.
+### Mapping
 
-### The Mapping
+The `opmphmMapping ()` uses your seed (the `OpmphmInit->seed` will be changed) and tries to map your
```

Comment:
I think we can drop the (), it somehow suggests that there would be no parameter. Maybe:
```
The function `opmphmMapping` 
```
is more clear?